### PR TITLE
Add integration tests for new /firefox (#7898)

### LINKED ISSUE #7898: Add functional tests for the new /firefox page
## Description

The redesigned page doesn't yet have any functional tests.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Page is covered in some way in our automated test runs.

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] Not having any coverage increases the risk of accidental breakage.

#### Tasks
- [ ] Write some tests.
- [ ] Push to `run-integration-tests` branch prior to merging to make sure they pass.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Test FB Container link only displays in firefox
- Test Menu List opens
- Add Menu List region

## Issue / Bugzilla link

Fix #7898 

## Testing
🏃‍♀ https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/446/pipeline/

### CODE REVIEW

File: tests/functional/conftest.py [hunk 1]
```diff
@@ -39,6 +39,13 @@ def internet_explorer(selenium):
     return selenium.capabilities.get('browserName').lower() == 'internet explorer'
 
 
+@pytest.fixture
+def ie9(selenium):
```
Comment (stephaniehobson) [inline tests/functional/conftest.py]: @alexgibson this skip is not triggered by ie9. Can you please have a look or recommend someone to help me with it?

Comment (alexgibson) [reply tests/functional/conftest.py]: You shouldn’t need to add any new code in order to remove a test from being run in IE9. Removing the `sanity` marker from the test should do it.

File: tests/functional/firefox/test_home.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.home import FirefoxHomePage
+
+
+@pytest.mark.sanity
```
Comment (alexgibson) [inline tests/functional/firefox/test_home.py]: Remove this line to not run the test in IE9

Comment (stephaniehobson) [reply tests/functional/firefox/test_home.py]: TY!

File: tests/pages/regions/menu_list.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+
+class MenuList(Region):
```
Comment (alexgibson) [inline tests/pages/regions/menu_list.py]: Nice work using a page region for this 👍

**Pull request merged**